### PR TITLE
zh: update TOC and remove empty files

### PR DESCRIPTION
**Pull request by toutdesuite**
### What is changed, added or deleted? (Required)

update TOC and remove empty files

### Which TiDB Data Migration (DM) version(s) do your changes apply to? (Required)

- [x] master (the latest development version, including v2.0 changes for now)
- [ ] v1.0 (TiDB DM 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: 
- Other reference link(s):

### CODE REVIEW

File: zh/TOC.md [hunk 1]
```diff
@@ -6,8 +6,7 @@
 ## 文档目录
 
 + 关于 TiDB Data Migration
-  + 基本信息
-    - [性能数据](performance.md)
+  + [性能数据](performance.md)
```
Comment (yikeke) [inline zh/TOC.md:9]: 这个链接到哪个文档

Comment (WangXiangUSTC) [reply zh/TOC.md:9]: https://github.com/pingcap/docs-dm/pull/85
这个 pr 里加了 performance-test.md，这个 performance.md 可以删了

Comment (yikeke) [reply zh/TOC.md:9]: ok，那先合 #85 再合 #95 吧

Comment (toutdesuite) [reply zh/TOC.md:9]: ok~

**Comment by miaoqingli** [comment]
LGTM

**Pull request merged**